### PR TITLE
recover password function

### PR DESCRIPTION
**Pull request by villacisJimmy**
- HTML file
- Send email function
- Confirm password

### CODE REVIEW

File: passwords.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mypass
```
Comment (villacisJimmy) [inline passwords.txt:1]: Looks good!, but could leave "mypass" as "my pass"

Comment (jota-villacis) [inline passwords.txt:1]: Looks good but could you edit "mypass" as "my pass"

**Review by jota-villacis — CHANGES_REQUESTED** [review]
Please update file passwords.txt

File: passwords.txt (hunk 1 above)
Comment (villacisJimmy) [reply passwords.txt]: The file is already edited, could you checked it again

**Review by jota-villacis — APPROVED** [review]
LGTM 🚀

**Pull request merged**